executor/src/witgen/jit/affine_symbolic_expression.rs

```diff
@@ -57,6 +59,32 @@ impl<T: FieldElement, V> Assertion<T, V> {
     }
 }
 
+pub enum LookupArgument<T: FieldElement, V> {
```

Comment:
```suggestion
pub enum MachineCallArgument<T: FieldElement, V> {
```
Let's move away from calling machine calls lookup :) It's wrong now (could be a permutation) and more wrong in when we move to bus interactions.